packages/venia-ui/lib/components/CheckoutPage/ItemsReview/item.css

```
@@ -0,0 +1,48 @@
+.root {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto auto 1fr;
+    column-gap: 1rem;
+    opacity: 1;
+    visibility: visible;
+    transition-duration: 350ms;
```

Venia's animation durations are generally in increments of 128ms. Half-increments are also acceptable for fine tuning. Here, I recommend slightly increasing the duration and adding a small delay to allow the container's reflow to complete before fading in.

```suggestion
    transition-delay: 64ms;
    transition-duration: 384ms;
```

---

Oh, good to know. Wasn't aware of that. Thank you.